blueocean-i18n/src/main/java/io/jenkins/blueocean/i18n/BlueI18n.java

```diff
@@ -136,12 +136,19 @@ public HttpResponse doDynamic(StaplerRequest request) {
     private JSONObject getBundle(BundleParams bundleParams, Locale locale) {
         PluginWrapper plugin = bundleParams.getPlugin();
 
-        if (plugin == null) {
-            return null;
-        }
 
         try {
-            ResourceBundle resourceBundle = ResourceBundle.getBundle(bundleParams.bundleName, locale, plugin.classLoader);
+            ResourceBundle resourceBundle;
+            if (plugin == null) {
+                Jenkins jenkins = Jenkins.getInstance();
+                if (jenkins!=null && jenkins.pluginManager!=null && jenkins.pluginManager.uberClassLoader!=null) {
```

Comment:
Actually we need it for the resource bundle which is stored in the classpath. Our common code is in a jar and contains the translations. Since it is not hpi we had to workaround that. 

Having said all this, we are trying another approach now.

Since we have our jsx in the common and create an instance like

```
Translator('commons')
```

However since this code will be imported to a real hpi plugin we could change it to something like

```
const callingPlugin = AppConfig.getCallerPlugin();
translator(callingPlugin)
```

And could revert the change to the java code.